templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="list_block">
+    <li>
+        <a href="{% url "taxi:index" %}">Home page</a>
+    </li>
+    <li>
+        <a href="#">Manufacturers</a>
```

Comment:
The link for 'Manufacturers' is a placeholder. Consider updating it with the actual URL using the `{% url %}` template tag when the view is available.